framework/src/executioners/Transient.C

```diff
@@ -62,7 +62,7 @@ Transient::validParams()
   params.addParam<Real>("start_time", 0.0, "The start time of the simulation");
   params.addParam<Real>("end_time", 1.0e30, "The end time of the simulation");
   params.addParam<Real>("dt", 1., "The timestep size between solves");
-  params.addParam<Real>("dtmin", 2.0e-14, "The minimum timestep size in an adaptive run");
+  params.addParam<Real>("dtmin", 4.0e-14, "The minimum timestep size in an adaptive run");
```

Comment:
Haha this seems like a pretty trivial change. I'd say that if we're going to change it, we should go an order of magnitude at least

Reply:
sounds good. Changed it